packages/core-web/src/styles/markbind.css

```diff
@@ -225,6 +220,11 @@ li.footnote-item:target {
     outline: none;
 }
 
+.popover-content {
+    overflow: auto;
+    max-height: 50vh;
+}
```

Comment:
I believe this css can be moved to `Popover.vue` now that there is a dedicated component for popovers. We can probably shift the css for the triggers as well to `Trigger.vue` but that can be in another PR.